modules/ROOT/pages/Development/BeginnersGuide/ReleaseMod.adoc

```diff
@@ -349,61 +349,83 @@ Consider asking on the Discord if you get stuck on this step.
 
 == Exporting
 
-Up until this point, you have likely been using the Alpakit 'copy to mods directory' feature to move files to the game's folders for testing.
+Up until this point, you have likely been using the Alpakit 'copy to mods directory' feature
+to stage your mods files in game's folders for testing.
 
-Using 'Alpakit Selected!' or the 'Alpakit!' (just this mod) buttons generate a zip archive for you located at
-// dedi-docs
-// Using 'Alpakit Dev' or the 'Alpakit!' (just this mod) buttons generate a zip archive for you located at
-`<starter project folder>/Saved/ArchivedPlugins/Windows/YourModReference/YourModReference-Windows.zip`.
+Behind the scenes, using the 'Alpakit Dev' or the 'Alpakit!' (just this mod) buttons
```

Comment:
Alpakit, not ALpakit

Reply:
![image](https://github.com/satisfactorymodding/Documentation/assets/47161774/e20e080c-651d-46eb-8525-ad46597a77f7)